packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -214,4 +220,40 @@ export class RedirectsMiddleware extends MiddlewareBase {
         })
       : undefined;
   }
+
+  /**
+   * When a user clicks on a link generated by the Link component from next/link,
+   * Next.js adds special parameters in the route called path.
+   * This method removes these special parameters.
+   * @param {string} pathname
+   * @param {string} queryString
+   * @returns {string} modified url
+   */
+  private getModifyURL(pathname: string, queryString: string) {
```

Comment:
Please, add more comments to explain the changes
Also add unit tests to cover new scenario